src/open_company_web/actions.cljs

```diff
@@ -419,4 +426,17 @@
     (do
       (cook/remove-cookie! :show-login-overlay)
       (dissoc db :show-login-overlay))
-    (assoc db :collect-name-password-error status)))
\ No newline at end of file
+    (assoc db :collect-name-password-error status)))
+
+(defmethod dispatcher/action :mobile-menu-toggle
+  [db [_]]
+  (assoc db :mobile-menu-open (not (:mobile-menu-open db))))
+
+(defmethod dispatcher/action :reset-su-list
+  [db [_]]
+  (let [slug (router/current-company-slug)
+        su-list-key (dispatcher/su-list-key slug)
+        latest-su-key (dispatcher/latest-stakeholder-update-key slug)]
+    (-> db
+      (update-in (butlast su-list-key) dissoc (last su-list-key))
```

Comment:
these 2 are expressions are fairly complicated, a comment here about what and why this is happening would be good.